rtcdtlstransport.go

```diff
@@ -31,13 +37,23 @@ type RTCDtlsTransport struct {
 	// OnError       func()
 
 	conn *dtls.Conn
+
+	srtpSession  *srtp.SessionSRTP
+	srtpEndpoint *mux.Endpoint
+
+	srtcpSession  *srtp.SessionSRTCP
+	srtcpEndpoint *mux.Endpoint
 }
 
 // NewRTCDtlsTransport creates a new RTCDtlsTransport.
 // This constructor is part of the ORTC API. It is not
 // meant to be used together with the basic WebRTC API.
 func (api *API) NewRTCDtlsTransport(transport *RTCIceTransport, certificates []RTCCertificate) (*RTCDtlsTransport, error) {
-	t := &RTCDtlsTransport{iceTransport: transport}
+	t := &RTCDtlsTransport{
+		iceTransport: transport,
+		srtpSession:  srtp.CreateSessionSRTP(),
```

Comment:
Maybe split this from the constructor so it'll be possible to start DTLS without SRTP (#346).

Reply:
Done!